cpp/unreal_plugins/SpCore/Source/SpCore/Unreal.h

```diff
@@ -546,6 +547,14 @@ class SPCORE_API Unreal
         return getItem(findActorsByClass<TReturnAsActor>(world, uclass));
     }
 
+    static bool attachToActor(AActor* child_actor, AActor* parent_actor) {
```

Comment:
I'd prefer to use the existing `UFUNCTIONS` in the Unreal Engine instead of creating these new entry points. Although the existing `UFUNCTIONS` are technically deprecated, they were deprecated a very long time ago (version 4.17) and don't seem to be going anywhere.